process/factory/interceptorscontainer/baseInterceptorsContainerFactory.go

```diff
@@ -100,7 +100,11 @@ func checkBaseParams(
 	if coreComponents.MinTransactionVersion() == 0 {
 		return process.ErrInvalidTransactionVersion
 	}
-	if check.IfNil(cryptoComponents.MultiSigner()) {
+	multiSigner, err := cryptoComponents.GetMultiSigner(0)
```

Comment:
no need for backwards compatibility flags because the node does not start, right?